src/main/java/com/welcome/bot/controllers/UserController.java

```diff
@@ -108,11 +139,15 @@ public UserSettings setUserSettings( @RequestBody UserSettings userSettings,@Cur
 
     	       
     	        User user = new User(signUpRequest.getEmail());
+    	       
     	        user.setRole(signUpRequest.getRole());
+    	        inviteRepository.save(user.getInvite());
+    	        userSettingsRepository.save(user.getUserSettings());
+    	        
     	        User result = userRepository.save(user);
     	        if(inviteService.sendInvite(result.getEmail())) {
     	        
-    	        user.getInvite().setSent(1);
+    	        result.getInvite().setSent(true);
```

Comment:
Instead of passing bool value, method can be defined as: 
`result.getInvite().delivered();`

Reply:
Resolved.